modules/migration-mtc-release-notes-1-8-5.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/mtc-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="migration-mtc-release-notes-1-8-5_{context}"]
+= {mtc-full} 1.8.5 release notes
+
+[id="technical-changes-1-8-5_{context}"]
+== Technical changes
+
+{mtc-first} 1.8.5 has the following technical changes:
+
+.Federal Information Processing Standard (FIPS)
+
+Federal Information Processing Standards (FIPS) are a set of computer security standards developed by the United States federal government in line with the Federal Information Security Management Act (FISMA).
+
+Starting with version {mtc-short} 1.8.5 is designed for FIPS compliance.
+
+[id="resolved-issues-1-8-5_{context}"]
+== Resolved issues
+
+{mtc-short} 1.8.5 has the following major resolved issues:
+// been told not to include CVEs in OCP operator release notes
+
+.Performing StorageClass conversion triggers the scale-down of all aplications of the namespace
+
+// will this be resolved for MTC 1.8.5
+In previous releases of {mtc-short}, when running a StorageClass conversion and if there was more than one application running, then {mtc-short} scaled down all the applications on the cutover phase, even those not involved in the migration. This issue has been resolved in {mtc-short} 1.8.5.  link:https://issues.redhat.com/browse/MIG-1661[(MIG-1661)]
+
+// there are only closed CVE-Jira tickets, do you want to keep the filter
+For a complete list of all resolved issues, see the list of link:https://issues.redhat.com/issues/?filter=12447122[{mtc-short} 1.8.5 resolved issues] in Jira.
+
+[id="known-issues-1-8-5_{context}"]
+== Known issues
+
```

Comment:
https://issues.redhat.com/browse/MIG-1660 

This can also be added to known issues

Reply:
See line 40 for MIG-1660